docs/user_management.md

```diff
@@ -0,0 +1,37 @@
+# How to give users the permissions to set up a run 
+Users that use the NPG pipeline could ask, through a ticket, to be 
+granted some permissions to set up runs. Permissions should be granted 
+only to those that are well known to the group. So, before adding users 
+to a permission group, it is better to ask the team (if that person is 
+unknown to you).
+
+There are following the steps to grant permissions on the 
+[SFWEB platform](http://sfweb.internal.sanger.ac.uk:9000/perl/npg):
```

Comment:
... to grant permissions using the tracking web pages. And delete the current text after 'permissions'